src/components/deploy/SupportPriceTable.js

```diff
@@ -290,7 +282,7 @@ export default () => (
     <SupportRow />
     <Divider className={styles.beyondDivider} />
     <BeyondRow />
-    <Footnotes />
+    <Footnotes footnotes={footnotes} />
```

Comment:
I'm seeing three footnote markers but only one footnote "Support via Slack..." etc.

Reply:
2 and 3 are described here

<img width="978" alt="image" src="https://user-images.githubusercontent.com/1479563/87833827-cc62ef80-c856-11ea-97a9-007209c89875.png">

Reply:
Ah, I see. I interpreted those as also referencing a footnote that I was expecting to be at the bottom of the page.

Reply:
Yeah good point, might be something to revisit in the future.

Reply:
Yeah, we can go ahead with this for now